src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySource.java

```diff
@@ -83,6 +84,8 @@ public final class BigQuerySource extends BatchSource<LongWritable, GenericData.
   private final BigQueryAvroToStructuredTransformer transformer = new BigQueryAvroToStructuredTransformer();
   // UUID for the run. Will be used as bucket name if bucket is not provided.
   private String bucketPath;
+  private String fqn = String.join(":", "bigquery",
```

Comment:
I don't think the Config object is initialized at this point. I think you should initialize this fqn during the `prepareRun` phase of the pipeline